curriculum/challenges/english/25-front-end-development/workshop-music-player/6752c62efa90554c01415e7b.md

```diff
@@ -0,0 +1,761 @@
+---
+id: 6752c62efa90554c01415e7b
+title: Step 38
+challengeType: 0
+dashedName: step-38
+---
+
+# --description--
+
+Next, you need to display the current song title and artist in the player display. Create a function named `setPlayerDisplay`. Inside the function, access the `#player-song-title` and `#player-song-artist` elements, responsible for displaying the song title and artist.
```

Comment:
isn't it better to access the two elemetns outside the function? otherwise we are creating them each time the function is called wastefully